### PR TITLE
Add custom word divider for TextBlock

### PR DESCRIPTION
**Pull request by miudit**
Some languages such as Japanese and Chinese has no "word divider" (the character between words).
Currently, when rendering Japanese in TextBlock, an unnecessary space is inserted between words.
jsfiddle: [https://jsfiddle.net/3ph9m0cx/](https://jsfiddle.net/3ph9m0cx/)

This PR makes it customizable.
In case of Japanese, setting `wordDivider = ""` resolves it.

### CODE REVIEW

**Comment by sebavan** [comment]
@darraghjburke could you double check this one ? adding @RaananW as well

**Review by RaananW — APPROVED** [review]
Great idea! I'll wait for @darraghjburke for the final approval

**Review by darraghjburke — APPROVED** [review]
LGTM! For future reference, could you provide a playground link showcasing the behavior rather than a JSFiddle? That would make it easier for us to test on the snapshot

**Pull request merged**
